services/py-api/src/database/repository/participants_repository.py

```diff
@@ -57,3 +58,10 @@ async def create(
         except Exception as e:
             LOG.exception("Participant insertion failed due to err {}".format(e))
             return Err(e)
+
+    async def get_number_random_participants(self):
+        return self._collection.find({"is_admin" : "false"}).count()
+        
+
+        
+    
```

Comment:
The spaces in the end of the file should be deleted. Normally, this should be done by the pre-commit hook. Remind me to see if you have it installed in your setup or not